docs/tools/offline/cli-device-management.md

```diff
@@ -0,0 +1,76 @@
+### Updating Devices with Mbed CLI
+
+Mbed CLI includes features that help prepare and ship updates for devices managed through Pelion. The Mbed CLI uses the subcommands starting with `mbed device-management`, `mbed dev-mgmt` or `mbed dm` to manage devices.
+
+Start by configuring your Mbed Cloud SDK API Key, target and toolchain. Obtain the API Key from the Pelion portal.
```

Comment:
For now, it's the Device Management Portal. Yes, this is a mouthful. :/

Reply:
API key (not Key)

Reply:
Start what? It would help if you explained the process you're going to detail.